docs/en/sql-reference/data-types/lowcardinality.md

```diff
@@ -0,0 +1,57 @@
+---
+toc_priority: 51
+toc_title: LowCardinality
+---
+
+# LowCardinality data type {#lowcardinality-data-type}
+
+Modifies other data types.
+
+## Syntax {#lowcardinality-syntax}
+
+```sql
+LowCardinality(data_type)
+```
+
+**Parameters**
+
+- `data_type` — Supported ClickHouse [data type](index.md).
+
+## Description {#lowcardinality-dscr}
+
+`LowCardinality` is a superstructure that changes a data storage method and rules of data processing. ClickHouse applies [dictionary coding](https://en.wikipedia.org/wiki/Dictionary_coder) to `LowCardinality`-columns. Operating with dictionary encoded data significantly increases performance of [SELECT](../statements/select/index.md) queries for many applications.
+
+Consider using `LowCardinality` instead of [Enum](enum.md) when working with strings. `LowCardinality` provides more flexibility in use and often reveals the same or higher efficiency.
+
+## Example
+
+Create a table with a `LowCardinality`-column:
+
+```sql
+CREATE TABLE lc_t
+(
+    `id` UInt16, 
+    `strings` LowCardinality(String)
+)
+ENGINE = MergeTree()
+ORDER BY id
+```
+
+## Related Settings and Functions
+
+Settings:
+
+- [low_cardinality_max_dictionary_size](../../operations/settings/settings.md#low_cardinality_max_dictionary_size)
+- [low_cardinality_use_single_dictionary_for_part](../../operations/settings/settings.md#low_cardinality_use_single_dictionary_for_part)
+- [low_cardinality_allow_in_native_format](../../operations/settings/settings.md#low_cardinality_allow_in_native_format)
+- [allow_suspicious_low_cardinality_types](../../operations/settings/settings.md#allow_suspicious_low_cardinality_types)
+
+Functions:
+
+- [toLowCardinality](../functions/type-conversion-functions.md#tolowcardinality)
+
+## See Also
+
+- [String Optimization (video presentation in Russian)](https://youtu.be/rqf-ILRgBdY?list=PL0Z2YDlm0b3iwXCpEFiOOYmwXzVmjJfEt). [Slides in English](https://github.com/yandex/clickhouse-presentations/raw/master/meetup19/string_optimization.pdf).
```

Comment:
Let's at least move Russian content to the bottom of the list, if not remove completely.